### DISCUSSION TITLE
韓國女團aespa 與Ai

### DISCUSSION THREAD

**Discussion by 109302003** [1101-GS4719-C/Python-Community-GS4719-C#264] [General]
https://uetie.com/tech/169004.html

有關注韓國女團就會知道爭議很多的女團aespa 每個人都有自己的AI人物，Aespa是一個超越現實與虛擬現實邊界的創新組合。Aespa的人類成員和她們的虛擬化身會一起合作舞台，與粉絲互動，豐富舞台。

**Comment by lyien1021** [comment]
好好奇真的有人會喜歡虛擬偶像嗎？我們為什麼不直接看真人就好🤔

**Reply by 109802503gloria** [reply]
我猜也許是可以確保不會崩人設吧，我身邊的朋友近距離與偶像互動後有幻滅，但我想虛擬的應該不會有這種問題

**Reply by 107602505** [reply]
有道理ㄟ通常粉絲都會對偶像有一些幻想但是出現一些負面新聞後可能就會失去熱情不再喜歡。

**Comment by JAIMEHUNG** [comment]
我覺得像lol推出的虛擬女團KDA比較有市場欸，因為有結合遊戲與KPOP，所以市場較豐富，但也敬佩科技能做的如此精細!!!!

**Comment by cody910120** [comment]
KDA超讚的！他們出的歌我還不聽爆

**Comment by mesakiyuta** [comment]
完全由AI製作出的虛擬偶像沒有人情味，感覺只是一個噱頭，未來應該會是往AI與實際偶像混合的方向發展。
